backends/dpdk/dpdkArch.cpp

```diff
@@ -2957,6 +2957,14 @@ MoveNonHeaderFieldsToPseudoHeader::addAssignmentStmt(const IR::Expression *e) {
 const IR::Node *MoveNonHeaderFieldsToPseudoHeader::postorder(IR::AssignmentStatement *assn) {
     if (is_all_args_header) return assn;
     auto result = new IR::IndexedVector<IR::StatOrDecl>();
+    if (isLargeFieldOperand(assn->left) && assn->right->is<IR::Constant>()) {
+        auto leftType = assn->left->type->to<IR::Type_Bits>()->width_bits();
+        auto rightType = assn->right->type->to<IR::Type_Bits>()->width_bits();
+        if (leftType == 128 && rightType == leftType) {
+            auto cst = assn->right->to<IR::Constant>();
+            if (!cst->fitsUint64()) return assn;
```

Comment:
how constant not fitting in uint64 will be handled?

Reply:
[backends/dpdk/dpdkHelpers.cpp] - This file has the changes in this same PR where Unsigned values more than 64 bit is handled.

Reply:
Why can the 128 bit case not be handled here? I am a little concerned that we might be masking failures here.

Also: the error message at line 2977 is incorrect and probably similar error messages, too. It looks like the DPDK backend only supports bit widths with power of 2, is that correct?

Reply:
Code generation for 128 bit constant is handled in dpdkHelpers.cpp since here only the constraints needed for the DPDK is checked in this file.  DPDK backend supports till 64 bits and assembly instructions to handle only 128 bit after 64 bits. Hence the intermediate number of bits are ignored